gst-plugin-pravega/src/pravegasink/imp.rs

```diff
@@ -281,6 +375,31 @@ impl ObjectImpl for PravegaSink {
                 None,
                 glib::ParamFlags::WRITABLE,
             ),
+            glib::ParamSpec::new_string(
+                PROPERTY_NAME_RETENTION_TYPE,
+                "Retention type",
+                "The retention policy for video stream. Can be days, size, daysAndBytes or none.",
```

Comment:
Change to:
If 'days', data older than 'retention-days' will be deleted from the stream. If 'bytes', the oldest data will be deleted so that the data size does not exceed 'retention-bytes'. If `daysAndBytes`, the oldest data will be deleted if it is older than `retention-days` or the data size exceeds `retention-bytes`.

Reply:
done